device-types/Lenovo/ThinkSystem-SR635.yaml

```diff
@@ -0,0 +1,17 @@
+---
+manufacturer: Lenovo
+model: ThinkSystem SR635
+slug: lenovo-thinksystem-sr635
+part_number: 7Y98
+u_height: 1
+is_full_depth: true
```

Comment:
To make sure we have the most complete information, can you add the following parameters, `airflow` `weight` and `weight_unit` with the required data